pkg/controller/service/servicecontroller.go

```diff
@@ -339,14 +339,23 @@ func (s *ServiceController) createLoadBalancerIfNeeded(namespacedName types.Name
 
 		// TODO: We could do a dry-run here if wanted to avoid the spurious cloud-calls & events when we restart
 
-		// The load balancer doesn't exist yet, so create it.
-		s.eventRecorder.Event(service, api.EventTypeNormal, "CreatingLoadBalancer", "Creating load balancer")
-
-		err := s.createLoadBalancer(service, namespacedName)
+		_, exists, err := s.balancer.GetLoadBalancer(s.loadBalancerName(service), s.zone.Region)
```

Comment:
does this just check that, eg: on gce we have a forwarding rule with that name, or does it verify that the forwarding rule is connected to a valid targetpool with the right nodes? if the former, the I *think* (didn't check code) the previous create would actually verify things are connected up properly.

Eitherway I think that's the desired behavior: if it doesn't exist create, if it does exist, verify. I'll take a todo if that never happened, but if it used to happen and it won't now we should address this some other way.